docs-source/content/faq/boot-identity-not-valid.md

```diff
@@ -0,0 +1,71 @@
+
+When you see these kinds of errors, it means that either the user name and password provided in the `weblogicCredentialsSecret` are invalid
+or that the WebLogic domain directory's security configuration files have changed in an incompatible way between when the Operator scanned
+the domain directory, which occurs during the "introspection" phase, and when the server instance attempted to start.
+
+First, always check that the user name and password credentials stored in the Kubernetes secret referenced by `weblogicCredentialsSecret` contain the expected values for an account with administrative privilege for the WebLogic domain.
+
+To understand the "incompatible domain security configuration" type of failure, it's important to review the contents of the
+[WebLogic domain directory](https://docs.oracle.com/middleware/12213/wls/DOMCF/config_files.htm#DOMCF140). Each WebLogic
+domain directory contains a "security" subdirectory that contains a file called "SerializedSystemIni.dat".  This file contains
+security data to bootstrap the WebLogic domain, including a domain-specific encryption key.
+
+During introspection, the operator generates a Kubernetes job that runs a pod in the domain's Kubernetes namespace and with the
+same Kubernetes service account that will be used later to run the Administration Server. This pod has access to the Kubernetes
+secret referenced by `weblogicCredentialsSecret` and encrypts these values with the domain-specific encryption key so that the
+secured value can be injected as the "boot.properties" contents when starting server instances.
+
+When the domain directory is changed such that the domain-specific encryption key is different, the "boot.properties" entries
+generated during introspection will now be invalid.
+
+This can happen in a variety of ways, depending on the [model selected](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/choosing-a-model/):
+
+#### Domain in a Docker image and rolling to an image containing new or unrelated domain directory
+
+The domain is using the "domain in a Docker image" model and the error occurs while rolling to a new Docker image that contains an entirely new or unrelated domain directory.
+
+The problem is that WebLogic cannot support server instances being part of the same WebLogic domain if the server instances do 
+not all share the same domain-specific encryption key and other bootstrapping security details. Additionally, operator introspection
+currently only happens when starting servers following a total shutdown. Therefore, the "boot.properites" generated from
```

Comment:
currently only happens -> currently happens only (Keep _only_ close to that which it modifies to improve the clarity and accuracy of the sentence.)